release-notes/Cloud/DCP_change_log.md

```diff
@@ -106,23 +106,28 @@ On the Catalog details page, if the publisher is not from your currently selecte
 
 On the Catalog details page, the side panel will now include a *Documentation* button to go to the external documentation.
 
-### 25 November 2024 - New feature - Catalog API - Service authenticated API [ID 41353]
+### 22 November 2024 - Fix - Catalog API - Registration with invalid manifest returned internal server error [ID 41516]
 
-The *api/service-catalog/v1-0/* route now exposes methods that allow applications to read from the Catalog using a "ServicePrincipal-JWT-Bearer".
+If you register a catalog item using a manifest that contained an invalid syntax for owner, a HTTP 500 internal server error was returned.
+This has been improved by returning a HTTP 400 Bad Request containing detail which field is invalid.
 
-This requires permission on the role "catalog.api.read" of the Catalog API.
+### 22 November 2024 - Fix - Catalog API - Registration with ID that exists in another organization returned internal server error [ID 41515]
 
-The following methods are now available:
+​If you register a catalog item with an ID that existed already in another organization, an internal server error was returned.
 
-- /catalogs/search
-- /catalogs/{CatalogId}
-- /catalogs/{CatalogId}/ranges
-- /catalogs/{CatalogId}/versions
-- /catalogs/{CatalogId}/versions/recommended
-- /catalogs/{CatalogId}/doc
-- /catalogs/categories
+This has been improved by returning a HTTP 409 Conflict result.
 
-You can make up to 5 requests per second, with occasional bursts of up to 20 requests allowed. After you trigger a burst, you will need to wait for the system to "refill" at a rate of 5 requests per second before you can trigger a burst again. Over a full minute, this allows for up to 300 requests.
+### 22 November 2024 - Feature - Catalog API - Get Catalog item categories [ID 41411]
+
+User, service and public Catalog APIs are extended with a categories call to obtain all categories supported in catalog
+
+"/api/user-catalog/v2-0/catalogs/categories"
+"/api/public-catalog/v2-0/catalogs/categories"
+"/api/service-catalog/v1-0/catalogs/categories"
+
+### 22 November 2024 - Feature - Catalog API - Update Catalog item publishing state using an organization key [ID 4191]
```

Comment:
The RN number for this item is incorrect. This one was originally included in #3954, but then got removed with the message that it would be included in a next release (see [https://github.com/SkylineCommunications/dataminer-docs/commit/b89d57c6f7e59f77828eb7847eaec1127cfc0e44](https://github.com/SkylineCommunications/dataminer-docs/commit/b89d57c6f7e59f77828eb7847eaec1127cfc0e44)).